### PR TITLE
test: add test for utilities

### LINKED ISSUE #69: Increase coverage
It's too low 54%.

### PR DESCRIPTION
**Pull request by juanpicado**
**Type:** test

increase coverage

Resolves #69

### CODE REVIEW

File: src/___tests___/__mocks__/Config.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import minimatch from 'minimatch';
 
 // FUTURE: we should use the same is on verdaccio
 export function getMatchedPackagesSpec(pkgName, packages) {
-  for (let i in packages) {
+  for (const i in packages) {
```
Comment (ayusharma) [inline src/___tests___/__mocks__/Config.js]:
```suggestion
  for (const pkg of packages) {
```

then return `pkg`

File: src/___tests___/utils.test.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import path from 'path';
+import { loadPrivatePackages } from '../utils';
+import logger from './__mocks__/Logger';
+
+describe('Utitlies', () => {
+  const loadDb = name => path.join(__dirname, '__fixtures__/databases', `${name}.json`);
```
Comment (ayusharma) [inline src/___tests___/utils.test.js]: can we move loadDb to outside describe block.

**Comment by juanpicado** [comment]
@ayusharma feel free to check again. I think 88% coverage is enough to do some refactoring later on.

**Comment by juanpicado** [comment]
I could go pretty far testing parameters recieved by each method, but I'd be here for ages 😆 . So, I just test the most common scenarios.

**Pull request merged**